doc/src/input/pairs.md

```diff
@@ -0,0 +1,166 @@
+$rc$:
+
+$$ V(r) = \begin{cases}
+    U(r) & r <= rc \\\\
+    0 & r > rc
+\end{cases}$$
+
+In the input, this uses a single string containing the cutoff radius as the
+`cutoff` value.
+
+```toml
+[global]
+cutoff = "10 A"
+
+[[pairs]]
+atoms = ["O", "O"]
+lj = {x0 = "3 A", k = "5.9 kJ/mol/A^2"}
+cutoff = "8 A"
+```
+
+### Shifted energy
+
+The potential $U$ can be additionally shifted to make sure it is continuous at
+$r = rc$. This is important for molecular dynamics, where a discontinuity means
+an infinite force in the integration.
+
+$$ V(r) = \begin{cases}
+    U(r) - U(rc) & r <= rc \\\\
+    0 & r > rc
+\end{cases}$$
+
+In the input, this uses a table containing the shifted value, which must be a
+string containing the cutoff radius.
+
+```toml
+[global]
+cutoff = {shifted = "8 A"}
+
+[[pairs]]
+atoms = ["O", "O"]
+lj = {x0 = "3 A", k = "5.9 kJ/mol/A^2"}
+cutoff = {shifted = "10 A"}
+```
+
+### Tail correction
+
+Tail corrections (also called long range corrections) are a way to add back the
+missing energy from using a cutoff distance. If the simulated system is
+homogeneous and isotropic after the cutoff (if the pair distribution function
+$g(r)$ is 1 after the cutoff), the missing energy for a potential $V(r)$ can be
+expressed using the density $\rho$:
+
+$$ U_\text{tail} = 2 \pi \rho \int_{rc}^{+\infty} r^2 V(r) \ dr $$
+
+In the input, this additional energetic term is controlled by the
+`tail_correction` keyword, which can be placed either in the `[global]` section,
+or in any specific `[[pairs]]` section.
+
+```toml
+# Use tail corrections for every pair interaction
+[global]
+tail_correction = true
+
+# Except for this one.
+[[pairs]]
+atoms = ["O", "O"]
+lj = {x0 = "3 A", k = "5.9 kJ/mol/A^2"}
+tail_correction = false
+```
+
+## Pairs restrictions
+
+Some force fields define additional restrictions concerning which particles
+should interact together. For example, sometimes bonded particles should not
+interact through electrostatic potential, or some interactions should only be
+taken in account for particles not in the same molecule. The way to specify this
+is to use restrictions. Restrictions can be used in two places: in the
+`[[pairs]]` section, and in the `[coulomb]` section. In both cases, they are
+specified with the `restriction` keyword, and one of the possible values.
+
+```toml
+[[pairs]]
+atoms = ["O", "O"]
+lj = {sigma = "3 A", epsilon = "123 kJ/mol"}
+restriction = {scale14 = 0.5}
+
+[coulomb]
+ewald = {cutoff = "8 A", kmax = 6}
+restriction = "intermolecular"
+```
+
+The possible values for `restriction` are:
+- `"intramolecular"` or `"intra-molecular"` to restrict the potential to only
+  particles in the same molecule;
+- `"intermolecular"` or `"inter-molecular"` to restrict the potential to only
+  particles NOT in the same molecule;
+- `"exclude12"` to exclude particles directly bonded together;
+- `"exclude13"` to exclude particles directly bonded together or forming an
```

Comment:
maybe use "and" instead of "or" to make clear that `"exclude13"` also contains `"exclude12"`?